packageRpm.nix

```diff
@@ -44,15 +74,16 @@ in rec {
         mkdir rpmbuild
         cd rpmbuild
         mkdir SPECS
+        mkdir SOURCES
         cp ${writeSpecFile} SPECS/${project}.spec
+        cp ${sourceArchive} SOURCES/${sourceArchive.name}
+        ls SOURCES
         mkdir -p BUILD/${project}
         cp ${licenseFile} BUILD/${project}/LICENSE
 
-        mkdir -p BUILDROOT/${pkgName}/usr/local/bin
-        cp ${bin} BUILDROOT/${pkgName}/usr/local/bin/${project}
-
-        rpmbuild -bb SPECS/${project}.spec
-        cp RPMS/${arch}/${pkgName}.rpm $out
+        rpmbuild -ba SPECS/${project}.spec
```

Comment:
What's the difference between `-ba` and `-bb`? Can you send some links which make it easier to understand what's going on here?

Reply:
-bb builds only binary package, -ba builds both binary and source packages